examples/platforms/utils/uart_rtt.h

```diff
@@ -0,0 +1,61 @@
+#ifndef UTILS_UART_RTT_H
+#define UTILS_UART_RTT_H
+
+#include "openthread-core-config.h"
+#include <openthread/config.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @def UART_RTT_BUFFER_INDEX
+ *
+ * RTT buffer index used for the uart.
+ */
+#ifndef UART_RTT_BUFFER_INDEX
+#define UART_RTT_BUFFER_INDEX 0
+#endif
```

Comment:
Could we add a static check to guarantee the LOG_RTT_BUFFER_INDEX and UART_RTT_BUFFER_INDEX won't be set to the same value?